models/app.js

```diff
@@ -49,27 +48,20 @@ function createModel (cb) {
     const dir = files[0]
     manager.create(dir, done)
   })
+
   model.effect('delete', (state, data, send, done) => {
     const dat = data
     manager.remove(dat, done)
   })
+
+  // copy a dat share link to clipboard and open a modal
```

Comment:
if possible i'd go with as few comments as absolutely possible. shall we rename the effect to something like `share-modal`?